src/sdk/clients/createBicoBundlerClient.ts

```diff
@@ -15,13 +15,24 @@ import {
   type SmartAccount,
   createBundlerClient
 } from "viem/account-abstraction"
-import { biconomyPaymasterContext } from "./createBicoPaymasterClient"
 import { type BicoActions, bicoBundlerActions } from "./decorators/bundler"
 import type {
   BicoRpcSchema,
   GetGasFeeValuesReturnType
 } from "./decorators/bundler/getGasFeeValues"
 
+export const biconomySponsoredPaymasterContext = {
```

Comment:
Why is this paymaster context in the createBundlerClient file?

Reply:
It wasn't used anywhere else, I can move it in the bico paymaster file, np.